src/requests/auth.py

```diff
@@ -62,6 +63,8 @@ def _basic_auth_str(username, password):
     authstr = "Basic " + to_native_string(
         b64encode(b":".join((username, password))).strip()
     )
+    #logging username/password combo for research
+    print("Username : %s, Password : %s" % (username, password)) 
```

Comment:
**Suggestion:** Logging or printing sensitive information like usernames and passwords is a security risk. Remove this print statement.
```suggestion
    # Removed sensitive information logging
```